.github/workflows/ci.yml

```diff
@@ -122,4 +122,4 @@ jobs:
         shell: bash
         run: >
           nix develop .#diffkemp-llvm${{ matrix.llvm }} --command bash -c
-          pytest tests -v
+          ${{ github.workspace }}/build/bin/run_e2e_tests.py -v --ignore='kernel/*'
```

Comment:
Is the `--ignore='kernel/*` necessary if in the `run_e2e_tests.py` file is running pytests for `tests` subfolder?

Reply:
Unfortunately yes, the other way could be changing the working directory in `run_e2e_tests.py`, which is up to discussion.

However, we have to handle it somehow, because otherwise the `pytest` is going to collect tests from `kernel/` directory and it is going to try to run them, which is going to result in failed build.

Reply:
When running pytest from CLI, we can do `pytest tests/` to tell pytest to look only in `tests/`. Isn't something similar possible using `pytest.main`?